ui/src/app/components/cases/list.component.html

```diff
@@ -198,7 +198,13 @@
           </a>
         </div>
         <div class="ts-col-15 px-25"
-             [textContent]="'testcase.status_'+testCase?.status| translate"></div>
+             [textContent]="'testcase.status_'+testCase?.status| translate">
```

Comment:
no fallback support required? since we use undefined check in status

Reply:
if so, have a computed property for getting test case status

Reply:
plz resolve this @nixonakash01

Reply:
that div will display the status of the testCase of whether its ready or etc. There is no possible values to be out of bounds for this field.

Reply:
testCaseStatus is a enum,  with these possible values DRAFT, IN_REVIEW, READY, OBSOLETE, REWORK.
@guganabu

Reply:
testCase?.status if so, we can remove ? check from the prop.

Reply:
.status returns the enum value. without it the value cannot be displayed